docs/wiki/[Examples]-Deploy-using-multiple-module-declarations.md

```diff
@@ -0,0 +1,806 @@
+<!-- markdownlint-disable first-line-h1 -->
+## Overview
+
+This page provides an example of how you could deploy your Azure landing zone using multiple declarations of the module.
+This covers scenarios such as:
+
+| Scenario | Description |
+| :--- | :--- |
+| Delegate responsibility using GitOps | Where an organization wants to use features such as CODEOWNERS to control who can approve changes to code for resources by category. |
+| Split across multiple state files | Due to the number of resources needed to deploy an Azure landing zone, some customers may want to split deployment across multiple state files. |
+| Simplify maintenance | Using multiple files to control the configuration of resources by scope makes it easier to understand the relationship to resources being managed by that code. |
+
+> **NOTE:**
+> This approach is very similar to the strategy described in [Deploy Using Module Nesting][wiki_deploy_using_module_nesting].
+
+This example builds on top of existing examples, including:
+
+- [Deploy Custom Landing Zone Archetypes][wiki_deploy_custom_landing_zone_archetypes]
+- [Deploy connectivity resources with custom settings][wiki_deploy_connectivity_resources_custom]
+- [Deploy Management Resources With Custom Settings][wiki_deploy_management_resources_custom]
+
+> **IMPORTANT:** Ensure the module version is set to the latest, and don't forget to run `terraform init` if upgrading to a later version of the module.
+
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/Azure/terraform-azurerm-caf-enterprise-scale?style=flat&logo=github)
+
+## Example root module
+
+For this example, we recommend splitting the code across the following files (grouped by folder):
+
+- root_module/
+  - [main.tf](#maintf)
+  - [variables.tf](#variablestf)
+  - modules/
+    - connectivity/
+      - [main.tf](#modulesconnectivitymaintf)
+      - [settings.connectivity.tf](#modulesconnectivitysettingsconnectivitytf)
+      - [variables.tf](#modulesconnectivityvariablestf)
+    - core/
+      - [main.tf](#modulescoremaintf)
+      - [settings.core.tf](#modulescoresettingscoretf)
+      - [settings.core.tf](#modulescoresettingsidentitytf)
```

Comment:
duplicate file name

Reply:
Good spot... Should be resolved now.

Have also made some other updates so PR ready for review again please.